dockerfiles/control/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04
+
+MAINTAINER ClusterHQ <contact@clusterhq.com>
+
+RUN \
+    apt-get --yes update \
+    && apt-get --yes install --no-install-recommends \
+        apt-transport-https \
+        ca-certificates \
+    && apt-get --yes clean
+
+ARG FLOCKER_VERSION="*"
+ARG FLOCKER_REPOSITORY="https://clusterhq-archive.s3.amazonaws.com/ubuntu/16.04/amd64"
+RUN echo "deb ${FLOCKER_REPOSITORY} /" > /etc/apt/sources.list.d/clusterhq.list
+RUN \
+    apt-get --yes update \
+    && apt-get --yes install --no-install-recommends --allow-unauthenticated \
+        clusterhq-python-flocker=${FLOCKER_VERSION} \
+        clusterhq-flocker-node=${FLOCKER_VERSION}
+
+EXPOSE 4523
+EXPOSE 4524
+
+ENTRYPOINT ["/usr/sbin/flocker-control", "--port=tcp:4523", "--agent-port=tcp:4524", "--journald", "--certificates-directory=/etc/flocker", "--data-path=/var/lib/flocker"] 
```

Comment:
What I tried is: 
1. I took dataset docker image and run a new container overriding the entry point and providing port mapping and mount points.
2. Then checked that it works by doing round-trip tests (moving volumes back and forth).
3. And finally I derived the Dockerfile from the docker-run parameters.

So I planed to check the Dockerfile when Jenkins will build the image, but as you pointed, it does not do it automatically, which explains why I could not find (docker-pull) it.